src/postgres-options.ts

```diff
@@ -15,7 +15,7 @@ export interface PostgresOptions<T> extends TransportStreamOptions {
     /**
      * Ssl active
      */
-    ssl?: boolean;
+    ssl?: {rejectUnauthorized :boolean } | boolean;
```

Comment:
Hello, just a small standard and I approve this PR
```suggestion
    ssl?: { rejectUnauthorized: boolean } | boolean;
```